woltka/ordinal.py

```diff
@@ -189,7 +189,7 @@ def read_gene_coords(fh, sort=False):
             # start and end are based on genome, not gene itself
             try:
                 start, end = sorted([int(x[1]), int(x[2])])
-            except IndexError:
+            except (IndexError, ValueError):
                 raise ValueError(line)
```

Comment:
This slipped my attention before, but it would be nice if you gave a helpful message here, not just `line`.

Reply:
Makes sense! Added "Cannot extract coordinates from line: " to error message.